edisgo/network/timeseries.py

```diff
@@ -2186,6 +2186,19 @@ def resample_timeseries(
         # set new timeindex
         self._timeindex = index
 
+    def scale_timeseries(
+        self, p_scaling_factor: float = 1.0, q_scaling_factor: float = 1.0
+    ):
+        attributes_type = ["generators", "storage_units", "storage_units"]
```

Comment:
"storage_units" are in here twice and I gues "loads" are missing?

Reply:
Oh that's bitter. I thought I tested it for all timeseries, but I didn't. I now added the missing assertions in the test.